docs/conf.py

```diff
@@ -26,6 +26,7 @@
 
 extensions = [
     'sphinx.ext.todo',
+    'sphinx_rtd_theme',
```

Comment:
Line needed according to the theme docs, so the localized strings are displayed correctly once our own translation contribution is merged :slightly_smiling_face: